components-core/src/main/java/com/adyen/checkout/components/core/PaymentMethodTypes.kt

```diff
@@ -94,6 +96,10 @@ object PaymentMethodTypes {
     const val WECHAT_PAY_QR = "wechatpayQR"
     const val WECHAT_PAY_WEB = "wechatpayWeb"
 
+    // Non-public payment method types only for internal use
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Not sure if this should be internal to the package itself since it doesn't fit in with this file but I'll leave it for you to decide

Reply:
Since this file is `public` it doesn't fit here, however it acts as a `txVariant` and used both while building the payment method object and getting the serializer. The `paymentMethod.type` field we get from `/paymentMethods` call is passed as the `brand` parameter, so we cannot use that. That's why among all the possible options this is the most meaningful place for this to be in.

Reply:
👍